docs/events/timeline-ui-overview.asciidoc

```diff
@@ -202,7 +202,25 @@ The {ref}/esql.html[Elasticsearch Query Language ({esql})] provides a powerful w
 
 You can use {esql} in Timeline by opening the **{esql}** tab. From there, you can: 
 
-- Explore your events using the default query, or create a custom one. The default query searches documents within the Security alert index (`.alerts-security.alerts-default`) and indices specified in the <<update-sec-indices,Security data view>>, then returns 10 events from the defined time range.  
+- Write an {esql} query to explore your events. For example, start with the following query, then iterate on it to tailor your results: 
++
+[source,esql]
+----
+FROM .alerts-security.alerts-default,apm-*-transaction*,auditbeat-*,endgame-*,filebeat-*,logs-*,packetbeat-*,traces-apm*,winlogbeat-*,-*elastic-cloud-logs-*
+| LIMIT 10
+| KEEP @timestamp, message, event.category, event.action, host.name, source.ip, destination.ip, user.name
+----
++
+This query does the following:
+
+** It starts by querying documents within the Security alert index (`.alerts-security.alerts-default`) and indices specified in the <<update-sec-indices,Security data view>>.
+** Then, the query limits the output to the top 10 results.
```

Comment:
Would this mean the top 10 most recent documents? Would that be worth mentioning?

Reply:
@michaelolo24 do you remember what the Discover folks said about this? [Here's](https://github.com/elastic/security-docs/pull/4065#discussion_r1374985875) the convo we had in 8.11 on this topic.

Reply:
For the record I did review the ES|QL [doc ](https://security-docs_bk_4578.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-commands.html#esql-limit) about LIMIT and it doesn't seem to say.

Reply:
Hey, yea. They don't actually guarantee it's the latest 10. They are randomly determined.

Reply:
Resolving this since we haven't doc'd this information in the past.